services/src/datasets/external/nature40.rs

```diff
@@ -332,8 +416,17 @@ impl MetaDataProvider<GdalLoadingInfo, RasterResultDescriptor, RasterQueryRectan
             }
         })?;
 
+        let (dataset, time_positions) = self.get_time_positions(dataset).await.unwrap();
+        let mut time: Option<TimeInterval> = None;
+
+        if let Some(time_index) = time_index_option {
+            let time_position = &time_positions[time_index];
+            let date_time = NaiveDateTime::parse_from_str(time_position, "%Y-%m-%dT%H:%M")?;
+            time = Some(TimeInterval::new_instant(TimeInstance::from(date_time))?);
+        }
+
         Ok(Box::new(GdalMetaDataStatic {
-            time: None,
+            time,
```

Comment:
- MetaDataProvider shouldn't return a *static* dataset
  - it should be a new type of GdalMetaData that stores the individual time positions
  - in the `loading_info` method it should use the `time_interval` of the `query` parameter to produce the correct loading info
    - imho this should mean gathering all time positions of the WCS dataset that are contained in the query `time_interval` and returning them as `GdalLoadingInfo`
	- the time position has to be injected into the WCS query in the `file_path` of the loading info
  - there are different ways to implement such a provider
    - a) like `GdalMetaDataStatic`, but instead of having a single config of the `GdalDatasetParameters` we would have a separate config for each time step
	- b) like `GdalMetaRegular`, but instead of specifying the time step between to rasters we would specify the valid time positions as an array

Reply:
A new GdalMetaData type for fixed time positions has been created.